src/inspector/network_agent.cc

```diff
@@ -171,12 +171,12 @@ std::unique_ptr<protocol::Network::Response> createResponseFromObject(
 
   protocol::String mimeType;
   if (!ObjectGetProtocolString(context, response, "mimeType").To(&mimeType)) {
-    return {};
+    mimeType = protocol::String("");
   }
 
   protocol::String charset = protocol::String();
   if (!ObjectGetProtocolString(context, response, "charset").To(&charset)) {
-    return {};
+    charset = protocol::String("");
```

Comment:
After merging with main, the parallel/test-inspector-network-content-type test was failing, so I made some changes.
I believe setting the default to an empty string shouldn't cause any issues.